### ISSUE TITLE
problem running on aws

### ISSUE THREAD

**Issue by kmulvey** [xinntao/Real-ESRGAN-ncnn-vulkan#28]
I downloaded the ubuntu zip on a g3s.xlarge and the result is a black image. Is the zip missing files?

```

./realesrgan-ncnn-vulkan-v0.2.0-ubuntu/realesrgan-ncnn-vulkan -i input.jpg -o out/output.jpg -n realesrgan-x4plus -s 4 

[0 Tesla M60]  queueC=0[16]  queueG=0[16]  queueT=1[2]
[0 Tesla M60]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Tesla M60]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/1
[0 Tesla M60]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1
[1 llvmpipe (LLVM 12.0.0, 256 bits)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[1 llvmpipe (LLVM 12.0.0, 256 bits)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 llvmpipe (LLVM 12.0.0, 256 bits)]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/0
[1 llvmpipe (LLVM 12.0.0, 256 bits)]  subgroup=8  basic=1  vote=1  ballot=1  shuffle=0
fopen /home/ubuntu/realesrgan-ncnn-vulkan-v0.2.0-ubuntu/models/realesrgan-x4plus.param failed
fopen /home/ubuntu/realesrgan-ncnn-vulkan-v0.2.0-ubuntu/models/realesrgan-x4plus.bin failed
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
0.00%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
4.17%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
8.33%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
12.50%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
16.67%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
20.83%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
25.00%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
29.17%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
33.33%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
37.50%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
41.67%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
45.83%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
50.00%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
54.17%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
58.33%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
62.50%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
66.67%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
70.83%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
75.00%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
79.17%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
83.33%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
87.50%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
91.67%
find_blob_index_by_name data failed
Try
find_blob_index_by_name output failed
Try
95.83%
```

**Comment by guss77** [comment]
The models are not available in the downloads for release 0.2.0.

I'm not sure how to properly get the models -  I got them by running the GTK app Upscaler from Flathub, and it was a non-trivial headache to do that. Also see issue #29

**Comment by guss77** [comment]
Yea, they are in another repo by the same dev, here: https://github.com/xinntao/Real-ESRGAN/releases
